test/extended/openstack/servers.go

```diff
@@ -182,22 +184,22 @@ func listWorkerMachineSets(ctx context.Context, dc dynamic.Interface) ([]objx.Ma
 }
 
 // getMachines lists all machines in the cluster
-func getMachines(ctx context.Context, dc dynamic.Interface) ([]objx.Map, error) {
+func getMachines(dc dynamic.Interface) ([]objx.Map, error) {
```

Comment:
Why do you remove the context argument in favour of calling `context.Background()` from within the function?